pkg/controller/manager/nad/controller.go

```diff
@@ -120,6 +131,23 @@ func (h Handler) OnChange(key string, nad *cniv1.NetworkAttachmentDefinition) (*
 	return nad, nil
 }
 
+func (h Handler) ensureLabels(nad *cniv1.NetworkAttachmentDefinition) error {
```

Comment:
what’s the purpose of this function?

Reply:
To add the label `network.harvesterhci.io/vlan-id"